tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project will actually build a `.dll` when we build the `.sln` file. That seems unnecessary. Maybe we should override the `Build` target or maybe not name it `.csproj` and instead just `.proj`? It's mission in life isn't to build `.cs` files into a `.dll`, but instead pull down external packages and provide the `GetContract` target.

Reply:
I tried renaming, but that did not work (it gave an error saying that project.assets.json was not generated.
I have also tried to overwrite the Build target, but I must have made some mistake since I still found the generated .dll.

What I did was adding:
 <Target Name="Build">
    <!-- This will override the default Build target. -->
  </Target>

to the .csproj file. 

What is the correct way to overwrite it?



---
In reply to: [280105436](https://github.com/dotnet/machinelearning/pull/3623#discussion_r280105436) [](ancestors = 280105436)

Reply:
Yes, I think that is fine.